network-api/networkapi/buyersguide/views.py

```diff
@@ -98,6 +98,9 @@ def category_view(request, slug):
         'category': category,
         'products': products,
         'mediaUrl': MEDIA_URL,
+        'pageTitle': pgettext(
+            'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+            '*privacy not included') + f' - {category}',
```

Comment:
I may me misremembering, but I recall `pgettext` only working with plain strings, so we may need to separate those two elements so we can update the template to use `{% block title%}{% trans "*privacy not included" %} - {{ page.name }}{% endblock %}` (where the category name will eventually be the right localized string once we finish the PNI wagtail uplift). At least, I think we had a block for this rather than setting `<title>` directly?

@TheoChevalier can you confirm? I think we ran into something similar a few weeks ago.